draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -403,7 +403,7 @@ For each claim that is to be selectively disclosed, the Issuer creates a Disclos
 The Issuer MUST create a Disclosure for each selectively disclosable claim as follows:
 
  * Create an array of three elements in this order:
-   1. A salt value. See (#salt-entropy) and (#salt_minlength) for security considerations. The salt value MUST be unique for each claim that is to be selectively disclosed. It is RECOMMENDED to base64url-encode the salt value, producing a string. Any other type that is allowed in JSON MAY be used, e.g., a number.
+   1. A salt value. See (#salt-entropy) and (#salt_minlength) for security considerations. The salt value MUST be unique for each claim that is to be selectively disclosed. It is RECOMMENDED to base64url-encode the salt value, producing a string. Any other type that is allowed in JSON MAY be used, e.g., a number. The salt MUST NOT be disclosed to third parties except the Holder.
```

Comment:
probably it is this sentence that would benefit from clarifying?: "The salt value MUST be unique for each claim that is to be selectively disclosed."
to something like "A new salt MUST be chosen for each claim independently from other salts." in a change below?

Reply:
Would you keep the other sentence that I added further below?